//package DayOne.Exception;
//
//public class ExampleOne {
//
//    try{
//        System.out.println("next statement");
//    } catch (Exception e){
//        System.out.println("nothing");
//    }
//    System.out.println("next statment");
//}
//    public static void main(String[] args) {
//
//        int numOne;
//        int numTwo;
//
//        try{
//            numOne = 0;
//            numTwo = 62/numOne;
//            String str = "TT";
//            System.out.println(str.charAt(2));
//            System.out.println(numTwo);
//            System.out.println("Inside Try catch block");
//        } catch (java.lang.ArithmeticException e){
//            System.out.println("number cannot be divided by zero");
//        } catch (NullPointerException e) {
//            System.out.println("String has null value");
//        } catch (StringIndexOutOfBoundsException e){
//            System.out.println("Index value is outside of string range.");
//        } catch (Exception e){
//            System.out.println("something went wrong");
//        }
//
//        System.out.println("outside of try catch block");
//    }
//}
